protos/format.proto

```diff
@@ -207,6 +227,25 @@ message Metadata {
   //   length = page_table[5][4][1];
   // ```
   uint64 page_table_position = 3;
+
+  // The schema of the statistics.
+  //
+  // This might be empty, meaning there are no statistics. It also might not 
+  // contain statistics for every field.
+  repeated Field stats_schema = 4;
```

Comment:
It is not clear to me what is the relationship between schema (which assumes a deep nested struct) and `stats_page_table`.

Reply:
Ah I guess we need a mapping between the field ids in the stats schema and their position in the page table?

Reply:
Added a field id list for statistics that maps from the schema to the page table.